indico/htdocs/js/presentation/Data/Remote.js

```diff
@@ -4,11 +4,6 @@
  */
```

Comment:
What is the advantage of changing these methods vs. just not using them and calling `$.ajax` directly instead? This is so legacy that we should just get rid of it if we have the chance to.

Reply:
RB is still using JSONRPC and while the operations like rejecting blockings could easily be changed, the few areas using `indicoSource` might be uglier to adapt. OTOH, I think it's used only in two places (rb holiday/weekend warning and room blocking), and replacing the former with a normal ajax call would be pretty easy and the latter could probably just be moved to a WTForm since it's a "standard" form

Reply:
Yeah, we'll get to RB soon anyway. No strong opinion, just wondering.